core/src/main/java/org/apache/struts2/dispatcher/HttpParameters.java

```diff
@@ -197,5 +197,28 @@ public HttpParameters build() {
 
             return new HttpParameters(parameters);
         }
+
+        /**
+        * Alternate Builder method which avoids wrapping any parameters that are already
+        * a {@link Parameter} element within another {@link Parameter} wrapper.
+        *
+        * @return 
+         */
+        public HttpParameters buildNoNestedWrapping() {
```

Comment:
~Looks good, but the name seems a bit awkward to me. I am not a native speaker but maybe `buildWithoutWrapping` or `buildUnwrapped` (or something else) would be a better name?~

Hm... after trying to find a better name, looks like `buildNoNestedWrapping` or longer `buildWithoutNestedWrapping` seems to be far better :)